lib/page/house_wikidata.rb

```diff
@@ -0,0 +1,31 @@
+require_relative '../../lib/popolo_helper.rb'
+
+module Page
+  class HouseWikidata
+    def initialize(slug, house)
+      @house = house
+      @slug = slug
+    end
+
+    def country
+      EveryPolitician.country(@slug)
+    end
+
+    def house
+      country[:legislatures].find { |h| h[:slug].downcase == @house.downcase }
```

Comment:
Now that you've made country be an `EveryPolitician::Country` that could just be `country.legislature(house)`